doc/sphinx/developer_manual/contributing_to_the_code/tips_and_tricks.md

```diff
@@ -4,4 +4,79 @@ Tips and tricks
 
 ```{todo}
 Give some tick and tricks, like start copying plugins which already so almost what you want.
-```
\ No newline at end of file
+```
+
+# Tricks to Run Tests
+
+## Trick 1: Run All Tests
+
+This command would compile and run all the tests.
+
+    make test
+
+Not so much information is given for the failing tests, so the next step is needed for more information.
```

Comment:
instead of `make test` you can do `ctest -V` to test directly get info on all tests. This may be a bit overwhelming though. (you can also do `ctest -VV` to get even more info, but I am not sure how useful that is.

Reply:
added that command.